### PR TITLE
feat(indexer): Exposed full list of StateChanges for a given block via view client

### PR DESCRIPTION
**Pull request by khorolets**
Add `get_state_changes_with_cause_in_block` method to store and `GetStateChangesWithCauseInBlock` for `ViewClient`.

Update Indexer Framework to return proper state changes
Bump Indexer Framework version to 0.7.0
Update Indexer Example with proper example structure according to recent changes.

## Testing Plan

* [x] NayDuck state changes tests pass
* [x] Indexer for Explorer is getting relevant data

### CODE REVIEW

**Review by frol — APPROVED** [review]

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
I run NayDuck since currently state changes are implemented in pytest suite: http://nayduck.eastus.cloudapp.azure.com:3000/#/run/812

**Comment by frol** [comment]
@bowenwang1996 we need your code owner approval for the view client changes

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**
